Rubberduck.Deployment/Writers/LocalDebugRegistryWriter.cs

```diff
@@ -30,12 +31,13 @@ public string Write(IOrderedEnumerable<RegistryEntry> entries)
                     throw new InvalidOperationException("Unexpected registry entry: " + entry.Key);
                 }
 
-                var value = ReplacePlaceholder(entry.Value, entry.Bitness);
-
-                var key = Registry.CurrentUser.CreateSubKey(entry.Key);
-                if (!(string.IsNullOrWhiteSpace(entry.Name) && string.IsNullOrWhiteSpace(value)))
+                if (Environment.Is64BitOperatingSystem)
+                {
+                    MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry64));
+                }
+                else 
                 {
-                    key.SetValue(entry.Name, value);
+                    MakeRegistryEntries(entry, Registry.CurrentUser);
```

Comment:
Not quite; we want to write to *both* 64-bit and 32-bit view on a 64-bit system. Because this is now in a `else` branch, it means only one or other are written to. Put it outside the `else` so that the 32-bit is always written to.

Reply:
I don't get it. Outside the else is where it was before. I think you mean something like, in the else then we call `MakeRegistryEntries` with implicit parameter, in the if, we call if twice with both RegistryViews explicitly. Right?